src/src/rx-serial/devSerialIO.cpp

```diff
@@ -7,13 +7,14 @@
 #include "SerialIO.h"
 #include "CRSF.h"
 #include "config.h"
-
 #define NO_SERIALIO_INTERVAL 1000
 
 extern SerialIO *serialIO;
 #if defined(PLATFORM_ESP32)
 extern SerialIO *serial1IO;
 #endif
+extern void setTargetSysId(uint8_t sysID);
+extern void setThisSysId(uint8_t sysID);
```

Comment:
I can take a look, we don't have RTTI so I will have to manually check if serial is appropriate type.

I tried to do it that way (with proper casting) and compiler complained a lot about unsafe casting and I don't want to enable RTTI as it ridiculously expensive in resources ROM and RAM.

Reply:
I've thought about this whole PR some more, and I think it would be better to add a new method to the superclass called `event` which has a default empty implementation.
The Mavlink class can then do whatever it needs to do. Then all the other stuff and unsafe casts will not be needed.

Reply:
Sounds good to me.

Can I keep the notes or do we have another place for that?

Reply:
I just don't like adding commented out code. It feels wrong to me. We should either delete it or implement it properly. If implementing Mavlink on the other serial port then it should be a different PR.

Reply:
Ok, I wo